rest/data/project_settings.go

```diff
@@ -410,9 +410,18 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 			return nil, errors.Wrap(catcher.Resolve(), "invalid periodic build definition")
 		}
 	case model.ProjectPageTriggersSection:
+		if !isRepo { // Check this for project refs only, as repo projects won't have last version information stored.
```

Comment:
I think this is not be needed since repo updates are routed through `SaveRepoSettingsForSection`

Reply:
Never mind